.vsts-ci.yml

```diff
@@ -345,3 +345,5 @@ extends:
               name: $(DncEngInternalBuildPool)
               image: 1es-windows-2022
               os: windows
+
```

Comment:
Ideally the VMR would not be dependent on a passing SDK build.  I can see this causing VMR breaks occasionally.  Is there a way to do this from a NuGet build instead?  Maybe even part of the sync process?

Reply:
The original ask as it came was that `nuget-client` should be validated by a flow through `sdk` where additional changes might be made to sdk to make things work. Only then these should flow together to the VMR.

Are you saying that this is not the flow we want?

Reply:
@premun and I discussed offline.  This was a misunderstanding on my part.  I was crossing my wires between how the code currently flows to the VMR on commit, versus how it will change with the flat flow on successful ci/darc publishing.